README.md

```diff
@@ -14,145 +14,91 @@ Under the covers matchmaking approaches touch on significant areas of computer s
+kubectl create namespace open-match
 
-```bash
-# Login to your Google Account for GCP.
-gcloud auth login
-gcloud config set project $YOUR_GCP_PROJECT_ID
-# Enable GCP services
-gcloud services enable containerregistry.googleapis.com
-gcloud services enable container.googleapis.com
-# Test that everything is good, this command should work.
-gcloud compute zones list
+# Install the core Open Match and monitoring services.
+kubectl apply -f https://github.com/GoogleCloudPlatform/open-match/releases/download/0.5.0-rc.1/install.yaml --namespace open-match
 ```
 
-Please follow the instructions to [Setup Local Open Match Repository](#local-repository-setup). Once everything is setup you can deploy Open Match by creating a cluster in Google Kubernetes Engine (GKE).
+### Deploy demo components
 
-```bash
-# Create a GKE Cluster and install Helm
-make create-gke-cluster push-helm
-# Deploy Open Match with example functions
-make REGISTRY=gcr.io/open-match-build TAG=0.4.0-e98e1b6 install-chart install-example-chart
-```
-
-To generate matches using a test client, run the following command:
+Open Match framework requires the user to author a custom match function and an evaluator that are invoked to create matches. For demo purposes, we will use an example MMF and Evaluator. The following command deploys these in the kubernetes cluster:
 
 ```bash
-make REGISTRY=gcr.io/open-match-build TAG=0.4.0-e98e1b6 run-backendclient
+# Install the example MMF and Evaluator.
+kubectl apply -f https://github.com/GoogleCloudPlatform/open-match/releases/download/0.5.0-rc.1/install-example.yaml --namespace open-match
 ```
 
-Once deployed you can view the jobs in [Cloud Console](https://console.cloud.google.com/kubernetes/workload).
+This command also deploys a component that continuously generates players with different properties and adds them to Open Match state storage. This is because a populated player pool is required to generate matches.
 
-### Local Repository Setup
+### Generate Matches! 
 
-Here are the instructions to set up a local repository for Open Match.
+The in a real setup, a game backend (Director / DGS etc.) will request Open Match for mathes. For demo purposes, this is simulated by a backend client that requests Open Match to continuously list matches till it runs out of players.
 
 ```bash
-# Install Open Match Toolchain Dependencies (for Debian, other OSes including Mac OS X have similar dependencies)
-sudo apt-get update; sudo apt-get install -y -q python3 python3-virtualenv virtualenv make google-cloud-sdk git unzip tar
-# Setup your repository like Go workspace, https://golang.org/doc/code.html#Workspaces
-# This requirement will go away soon.
-mkdir -p $HOME/workspace/src/github.com/GoogleCloudPlatform/
-cd $HOME/workspace/src/github.com/GoogleCloudPlatform/
-export GOPATH=$HOME/workspace
-export GO111MODULE=on
-git clone https://github.com/GoogleCloudPlatform/open-match.git
-cd open-match
+# Install the example MMF and Evaluator.
```

Comment:
This line is wrong

Reply:
Removed it now - the explanation above seems sufficient.